.buildkite/premerge.steps.yaml

```diff
@@ -35,10 +35,11 @@ steps:
   # New build pipeline
   # Trigger a 4.26 build
   - trigger: "unrealgdkbuild-ci"
-    label: "testgyms-ci-426 ${BUILDKITE_MESSAGE}"
-    async: true
+    label: "testgyms-ci-4.26"
```

Comment:
are dots ok?

Reply:
Confirmed they are ok. Gives us consistency throughout the repos now too.